### PR TITLE
feat: adds quiz questions chore: adds answers to yml

### PR DESCRIPTION
**Pull request by RichHawkins**

### CODE REVIEW

File: lesson_03/quiz/lesson_03_quiz/src/test/resources/application.yml [hunk 1]
```diff
@@ -3,6 +3,10 @@ spring:
     active: dev
 quiz:
   answers: 
+    richhawkins:
+     - $2y$10$RXLPI31gEjCF1tkZ.nS20.sPOb9SFXrPeVZv0Ezg.YD/hlcAVpDoK
+     - $2y$10$27WkeXYw4lCbtqCGFh/afeX75BoSTaLVsL6eSKT8CJgcDE.z//mH2
+     - $2y$10$OaEPPpERWwAkNghJg6tzBuopXjTwDhu3knf.DNYAJ2OjaRsKfBU/2
```
Comment (anthonydmays) [inline lesson_03/quiz/lesson_03_quiz/src/test/resources/application.yml:9]: You should only have two answers specified since you have two questions.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

File: lesson_03/quiz/lesson_03_quiz/src/main/java/com/codedifferently/lesson3/quizzes/RichHawkinsQuiz.java
Comment (anthonydmays) [inline lesson_03/quiz/lesson_03_quiz/src/main/java/com/codedifferently/lesson3/quizzes/RichHawkinsQuiz.java:1]: Need to fix formatting on this file.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

File: lesson_03/quiz/lesson_03_quiz/src/main/java/com/codedifferently/lesson3/quizzes/RichHawkinsQuiz.java
Comment (RichHawkins) [reply lesson_03/quiz/lesson_03_quiz/src/main/java/com/codedifferently/lesson3/quizzes/RichHawkinsQuiz.java:1]: ran spotlessApply again, format should now be correct.

File: lesson_03/quiz/lesson_03_quiz/src/main/java/com/codedifferently/lesson3/quizzes/AnthonyMaysQuiz.java
Comment (anthonydmays) [inline lesson_03/quiz/lesson_03_quiz/src/main/java/com/codedifferently/lesson3/quizzes/AnthonyMaysQuiz.java:1]: Please revert changes to this file.

Comment (RichHawkins) [reply lesson_03/quiz/lesson_03_quiz/src/main/java/com/codedifferently/lesson3/quizzes/AnthonyMaysQuiz.java:1]: reverted file to original

**Review by anthonydmays — APPROVED** [review]
LGTM

**Pull request merged**